ftest/src/test/java/org/jboss/arquillian/graphene/ftest/element/EventHandler.java

```diff
@@ -0,0 +1,33 @@
+/*package org.jboss.arquillian.graphene.ftest.element;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.events.WebDriverEventListener;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+
+import org.jboss.arquillian.graphene.guard.RequestGuardFactory;
+
+public class EventHandler implements WebDriverEventListener {
```

Comment:
What was the purpose of it?

Reply:
To track the following interactions: clicking, sending keys to an input box and clearing an input box. Write them to static storage and check it when calling .verify() on an actions object, confirming the above actions